src/main/java/com/forest/mytopmovies/config/swagger/SwaggerConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.forest.mytopmovies.config.swagger;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SwaggerConfig {
+    @Bean
+    public OpenAPI customOpenAPI() {
+        return new OpenAPI()
```

Comment:
Refactor this to use methods to create components and security scheme, it will make it easier to read.